client/src/Movies/MovieList.js

```diff
@@ -32,23 +34,14 @@ export default class MovieList extends Component {
 }
 
 function MovieDetails({ movie }) {
-  const { title, director, metascore, stars } = movie;
+  
   return (
-    <div className="movie-card">
-      <h2>{title}</h2>
-      <div className="movie-director">
-        Director: <em>{director}</em>
+    
+      <div>
+      <Link to ={`/movies/${movie.id}`}>
```

Comment:
Usually, it's not a good convention to wrap the component in `Link` , try to wrap the movie card either inside of the component (inside your return) or inside your map function